app/assets/stylesheets/contact.scss

```diff
@@ -11,6 +11,12 @@ body {
 .contact-form-text-area {
   color: black;
 }
+.button-text-area {
+  color: black;
+}
+.form-dropdown {
+  color: black;
+}
```

Comment:
This sort of stuff I'm sure Caleb will completely change in the future, but this is a quick fix to make the site more usable for now